### PR TITLE
[MERGE WITH Frontend] New literature request fixes

### PR DESCRIPTION
**Pull request by ntarocco**
- reviewed marshmallow loaders
- rename REJECTED state to DECLINED to use a more polite term

**WARNING**

1. To be merged with frontend one: https://github.com/inveniosoftware/react-invenio-app-ils/pull/360
2. Demo/Migrated data has to be recreated after local install or env deployment

### CODE REVIEW

**Review by jrcastro2 — APPROVED** [review]

File: invenio_app_ils/config.py [hunk 1]
```diff
@@ -927,18 +927,6 @@ def _(x):
 ILS_INDEXER_TASK_DELAY = timedelta(seconds=2)
 """Trigger delay for celery tasks to index referenced records."""
 
-# Accounts REST
-# ==============
-ACCOUNTS_REST_READ_USER_PROPERTIES_PERMISSION_FACTORY = backoffice_permission
```
Comment (KonstantinaStoikou) [inline invenio_app_ils/config.py:932]: Why are those removed?

**Review by KonstantinaStoikou — APPROVED** [review]

File: invenio_app_ils/config.py (hunk 1 above)
Comment (ntarocco) [reply invenio_app_ils/config.py:932]: Well spotted! I think this is a left over of when @kprzerwa was working on `accounts-rest` module but we abandoned it.
@kprzerwa can you please check?

Comment (kpsherva) [reply invenio_app_ils/config.py:932]: I don't think it was me working on it, I think it was Zach :) but I checked, we don't use invenio-accounts-rest I think it is OK to remove it

File: invenio_app_ils/document_requests/loaders/jsonschemas/document_request.py [hunk 2]
```diff
@@ -53,15 +54,21 @@ class Meta:
     issn = SanitizedHTML()
     issue = SanitizedHTML()
     journal_title = SanitizedHTML()
+    legacy_id = fields.Str()
```
Comment (kpsherva) [inline invenio_app_ils/document_requests/loaders/jsonschemas/document_request.py:57]: is it added because of disappearing legacy_id on update action ?

**Review by kpsherva — APPROVED** [review]
